src/ngraph/runtime/intelgpu/intelgpu_op_custom_kernels.cpp

```diff
@@ -204,18 +204,17 @@ void runtime::intelgpu::do_pad_operation(cldnn::topology& topology,
     vector<size_t> gws;
 
     // The kernel name and parameters
-    runtime::intelgpu::gen_func_def(
-        writer, entry_point_name, {2, "float"}, {input_shape, {1}}, "float", output_shape);
+    gen_func_def(writer, entry_point_name, {2, "float"}, {input_shape, {1}}, "float", output_shape);
```

Comment:
It is better to do refactoring like this, removing namespace, in a separate PR. All this does is add deltas that are unrelated to this PR.